apps/console/src/features/organizations/components/edit-organization-role/edit-organization-role.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```

Reply:
Resolved in https://github.com/wso2/identity-apps/pull/3150/commits/4881fb75f0c759b0224acf6e192dc4addb2e9cf4